docs/src/components/markdown.astro

```diff
@@ -3,14 +3,13 @@ import { asHtml } from "@/lib/as-html";
 
 type Props = {
   readonly markdown?: string;
+  readonly className?: string;
 };
 
-const { markdown } = Astro.props;
+const { markdown, className } = Astro.props;
 ---
 
 <article
-  class="prose prose-slate dark:prose-invert prose-code:text-primary prose-code:before:content-none prose-code:after:content-none"
+  class={`prose prose-slate dark:prose-invert prose-code:text-primary prose-code:before:content-none prose-code:after:content-none ${className ?? ""}`}
   set:html={markdown === undefined ? undefined : asHtml(markdown)}
->
-  <slot />
```

Comment:
I'm confused, how does this work if you removed the slot? We use the slotted approach for the migration boxes...

Reply:
`set:html` is the thing that does this here; this was apparently against the https://ota-meshi.github.io/eslint-plugin-astro/rules/no-conflict-set-directives/ rule

Reply:
but doesn't this mean we need to send the markdown as a param instead of a child in the migration box?

Reply:
huh yeah, i checked and i guess they're both needed????